sympy/printing/pretty/pretty_symbology.py

```diff
@@ -208,8 +208,9 @@ def xstr(*args):
     'hat': lambda s: s+u('\N{COMBINING CIRCUMFLEX ACCENT}'),
     'bar': lambda s: s+u('\N{COMBINING OVERLINE}'),
     'vec': lambda s: s+u('\N{COMBINING RIGHT ARROW ABOVE}'),
-    'prime': lambda s: s+u(' \N{COMBINING VERTICAL LINE ABOVE}'),
-    'prm': lambda s: s+u(' \N{COMBINING VERTICAL LINE ABOVE}'),
+    # alternatively: MODIFIER LETTER PRIME or even plain old APOSTROPHE
+    'prime': lambda s: s+u('\N{PRIME}'),
+    'prm': lambda s: s+u('\N{PRIME}'),
```

Comment:
With Unicode, it's generally advisable to stick with the character that is what you wanted, not just something that looks similar. Mostly because fonts tend to vary symbol display according to the stated purpose, so misselections may look "odd" on different operating systems. If the proper symbol looks odd, that may be because your font has a bug (fonts contain code), and often those bugs are fixed in later versions of the font file so it may be better to stick with the oddness for now.

This rules out APOSTROPHE.
Wikipedia says that MODIFIER LETTER PRIME is supposed to be used for linguistic purposes, so that's ruled out too; the comment can be removed.

Also, this makes me 100% +1 on using `\N{PRIME}`.